src/app/core/store/shopping/product-listing/product-listing.effects.ts

```diff
@@ -165,8 +167,8 @@ export class ProductListingEffects {
       map(({ id, filters }) => ({ type: id.type, value: id.value, filters })),
       distinctUntilChanged(isEqual),
       // TODO: (Sparque handling) temporary solution until the category navigation will be handled by Sparque
```

Comment:
is this ToDo still relevant?

Reply:
yes will removed with sparque navigation feature branch